packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -77,7 +77,7 @@ export class VaAccordion {
         details: {
           header: headerText || clickedItem.header,
           subheader: clickedItem.subheader,
-          level: headerLevel || clickedItem.level,
+          level: headerLevel !== undefined ? parseInt(headerLevel) : headerLevel || clickedItem.level,
```

Comment:
Comes back as text when using getSlottedNodes and pulling from the tagName so still need to convert to Int to match format of level props which is Int

Reply:
@k80bowman that's correct. I really just moved the parseInt() from getSlottedNodes into the analytics event because that is where the data is used, and `clickedItem.level` gives us an Integer so I'm trying to keep the types given to the Analytics event the same. When we pull from getSlottedNodes we get the DOM Node back and then I'm parsing through the tagName to get us the heading level which would be 1-6 but doing that gives us that number as a string instead of an integer and works in all other browsers but IE11. In IE11 that parsing for the tagName via getSlottedNodes returns an `EmptyTextNode` so if we are in IE11 we are now doing an optional conditional which would return `undefined` and then `clickedItem.level` instead.
So it boils down to this:
* Modern browser where someone uses `slot="headline"` we need to change the values type from a string to an integer
* IE11 where someone uses `slot="headline"` tagName property doesn't exist therefore return `undefined` which would then fall to the second value which is `clickedItem.level`
* All Browsers (Modern and IE11) someone sets the level directly like: `<va-accordion-item header="Second Amendment" level={5} id="second">` then `clickedItem.level` will be used

Reply:
I'm not sure this is necessary. If we call `parseInt` on an undefined value we just get `NaN`, it doesn't throw an error:

![image](https://user-images.githubusercontent.com/2008881/152248136-c7216075-c8fd-4b5c-b602-67e7038aa9dc.png)

Is something like this possible?

```js
      headerLevel = parseInt(node.tagName?.toLowerCase().split('')[1]);
      
      // ...
                level: headerLevel || clickedItem.level,
```

The expression `NaN || 3` should evaluate to 3 so I think we would be covered.

Reply:
@bkjohnson I reverted the parseInt() location back as suggested and pushed up those changes